core/src/main/java/tech/tablesaw/io/FileReader.java

```diff
@@ -129,15 +129,16 @@ public String[] getColumnNames(
    * @param headerNames The header names to be potentially adjusted.
    */
   private void renameDuplicateColumnHeaders(String[] headerNames) {
-    Map<String, Integer> nameCounter = new HashMap<>();
+    final Map<String, Integer> nameCounter = new HashMap<>();
```

Comment:
Let's remove `final` everywhere. It doesn't match the style of the codebase

Maybe we can consider whether to migrate to `var` after a release has been out for awhile if folks don't complain about us dropping Java 8. Perhaps `final var` would look less noisy than `final` looks with types. The other option might be lombok's `val`, but not sure if I'd want to introduce something that non-standard